src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs

```diff
@@ -29,10 +31,31 @@ public CalendarEventsController(ILogger<CalendarEventsController> logger, IMedia
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> Get(Guid calendarEventId, [FromHeader(Name = Constants.RequestHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId)
     {
-        _logger.LogInformation("AAN Hub API: Received command from User ID {requestedByMemberId} to get calendar event by event ID {calendarEventId}", requestedByMemberId, calendarEventId);
+        _logger.LogInformation("AAN Hub API: Received command from User ID {requestedByMemberId} to get calendar event by event ID {calendareventid}", requestedByMemberId, calendarEventId);
```

Comment:
Why change the case here? This is only for logging attributes.

Reply:
Reverted